### ISSUE TITLE
Unable to drag D-Lite down

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#4714]
Currently it is not possible to move D-Lite down by dragging from the top area. See this video:

[![Image from Gyazo](https://i.gyazo.com/4b66faef10b6d89882c56bed3f882c9f.gif)](https://gyazo.com/4b66faef10b6d89882c56bed3f882c9f)

**Comment by IanMayo** [comment]
@helenayele - does it work if you move D-Lite down the screen a little? (I'm just wondering if it only happens when D-lite is at the top of your screen)

**Comment by helenayele** [comment]
Yes,  I understood why it was not working for me. The reason is that , I maximized it before I drag it.  If you first maximize it, then you can't drag it down unless you restore it. But it shouldn't behave like this, for example if I maximize Eclipse IDE and try to drag down, it will immediately restore to some width/height.
[![Image from Gyazo](https://i.gyazo.com/538f3c7934491e5302a395895f5f8bd2.gif)](https://gyazo.com/538f3c7934491e5302a395895f5f8bd2)

**Comment by IanMayo** [comment]
@helenayele - so is this a MS Windows layout manager issue?

**Comment by helenayele** [comment]
I don't think it is operating system issue. B.c products like Eclipse works fine while resizing, moving,etc.
But D-Lite stacks when I try to drag it down very often

**Comment by helenayele** [comment]
May be I am not clear with your question. But when I try to drag Eclipse IDE while it is in maximize state, dragging just works. But D-Lite is not flexible to this operation.

**Comment by ayeshare** [comment]
@helenayele @IanMayo I didnt even know such a feature existed in windows.  Older windows this was not possible.  Assigning to myself

**Comment by helenayele** [comment]
@ayeshare  would you please consider fixing this issue?

**Comment by IanMayo** [comment]
@helenayele & @ayeshare - let's keep this open, but let's keep it as a lower priority.  I see it as a "nice to have", for once more important issues are completed.
